composer.json

```diff
@@ -20,7 +20,8 @@
         "symfony/framework-bundle": "~2.0"
     },
     "require-dev":{
-        "symfony/console": "~2.0"
+        "symfony/console": "~2.0",
+        "symfony/expression-language": "~2.6"
```

Comment:
I checked and at least for 2.3 it seemed to install FrameworkBundle 2.3 .. wasn't sure if there is a more elegant way to handle the issue.